source/_integrations/habitica.markdown

```diff
@@ -145,3 +145,10 @@ Also an event `habitica_api_call_success` will be fired with the following data:
     "id": "NEW_TASK_UUID"}
 }
 ```
+
+This Markdown will list the tasks saved in `sensor.habitica_USER_todos.attributes.items` in a card.
```

Comment:
- Maybe add a heading? Right now, it feels completely out of context.
- This isn't Markdown, it is a template, which can be used in a Markdown card. Maybe make that more clear (and link to the Markdown card documentation?)